transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -201,7 +201,8 @@ static jboolean netty_io_uring_probe(JNIEnv *env, jclass clazz, jint ring_fd, ji
 
     jsize opsLen = (*env)->GetArrayLength(env, ops);
     jint *opsElements = (*env)->GetIntArrayElements(env, ops, 0);
-    for (int i = 0; i < opsLen; i++) {
+    int i;
```

Comment:
why is the int declaration outside?

Reply:
because the gcc we use on our docker image will fail otherwise as it uses an other c standard as default which not supports this.